trafficserver_exporter/collector.py

```diff
@@ -480,10 +480,11 @@ def parse_metrics(self, data):
             'trafficserver_ram_cache_hits_total',
             'RAM cache hit count.',
             'counter')
-        metric.add_sample(
-            'trafficserver_ram_cache_hits_total',
-            value=data['proxy.process.cache.ram_cache.hits'],
-            labels={'volume': str(volume)})
+        if len(volumes):    #Only add if we have volume metrics
```

Comment:
Just a small style nit here (trying to follow PEP8), use two spaces after the code, followed by one after the comment, like
`if len(volumes):  # comment`.

I'm using flake8 (https://pypi.python.org/pypi/flake8) for linting.
If you `pip install tox` you can just run `tox` in the root of the repo, or just flake8 with `tox -e pep8`